src/renderer/api/__tests__/pods.test.ts

```diff
@@ -166,14 +158,13 @@ function getDummyPod(opts: GetDummyPodOptions = getDummyPodDefaultOptions()): Po
 describe("Pods", () => {
   const podTests = [];
 
-  for (let r = 0; r < 3; r += 1) {
-    for (let d = 0; d < 3; d += 1) {
-      for (let ir = 0; ir < 3; ir += 1) {
-        for (let id = 0; id < 3; id += 1) {
-          podTests.push([r, d, ir, id]);
-        }
-      }
-    }
+  for (let r = 0; r < 100; r += 1) {
```

Comment:
I think Jari changed this test to range over 3 instead of 10 per for loop, so this PR is now actually adding more tests :)

Reply:
It is, but not nearly to the same extent.

Reply:
Plus 3^4 is pretty close (from an order of magnitude POV) to 10^2

Reply:
I'm confused. Is sparse coverage over a higher value set (0->50)  a better test set than full coverage over  (0->3)? The functionality being tested isn't really dependent on how big n is in (0->n), is it?

Reply:
For certain types of tests I believe that is that case. This way you can have a greater coverage area.

I acknowledge that it doesn't prevent people from just rerunning the tests until they pass though.